src/lib/gundb/UserStorageClass.js

```diff
@@ -26,7 +26,7 @@ import Config from '../../config/config'
 import API from '../API/api'
 import pino from '../logger/pino-logger'
 import isMobilePhone from '../validators/isMobilePhone'
-import resizeBase64Image from '../utils/resizeBase64Image'
+import { resizeImage } from '../utils/image'
```

Comment:
@mauriguz @MathiasMoralesDixtra please use meaningful commit descriptions
this is about image resize and you commited something about text alignment

Reply:
Sorry, I confused the commit message.